### PR TITLE
New resource: `azurerm_automation_hybrid_runbook_worker`

### PR DESCRIPTION
**Pull request by wuxu92**
add support for Automation RunbookWorker.

=== RUN   TestAccHybridRunbookWorker_basic
=== PAUSE TestAccHybridRunbookWorker_basic
=== CONT  TestAccHybridRunbookWorker_basic
--- PASS: TestAccHybridRunbookWorker_basic (826.32s)
PASS

### CODE REVIEW

File: internal/services/automation/automation_hybrid_runbook_worker.go [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+			ForceNew:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+		"worker_id": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.IsUUID,
+		},
+		"vm_resource_id": {
```
Comment (katbyte) [inline internal/services/automation/automation_hybrid_runbook_worker.go:57]: could we format this like most of the provider is
```suggestion
		"resource_group_name": commonschema.ResourceGroupName(),
		
		"automation_account_name": {
			Type:         pluginsdk.TypeString,
			Required:     true,
			ForceNew:     true,
			ValidateFunc: validation.StringIsNotEmpty,
		},
		
		"worker_group_name": {
			Type:         pluginsdk.TypeString,
			Required:     true,
			ForceNew:     true,
			ValidateFunc: validation.StringIsNotEmpty,
		},
		
		"worker_id": {
			Type:         pluginsdk.TypeString,
			Required:     true,
			ForceNew:     true,
			ValidateFunc: validation.IsUUID,
		},
		
		"vm_resource_id": {
```

File: internal/services/automation/automation_hybrid_runbook_worker.go [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+	}
+}
+
+func (m HybridRunbookWorkerResource) Attributes() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"ip": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"registered_date_time": {
```
Comment (katbyte) [inline internal/services/automation/automation_hybrid_runbook_worker.go:72]: anbd so forth

File: website/docs/r/automation_hybrid_runbook_worker.html.markdown [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+* `vm_resource_id` - (Required) The ID of the virtual machine used for this HybridWorker. Changing this forces a new Automation to be created.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Automation Hybrid Runbook Worker.
+
+* `ip` - The IP address of assigned machine .
```
Comment (katbyte) [inline website/docs/r/automation_hybrid_runbook_worker.html.markdown:45]:
```suggestion
* `ip` - The IP address of assigned machine.
```

File: website/docs/r/automation_hybrid_runbook_worker.html.markdown [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+* `last_seen_date_time` - Last Heartbeat from the Worker.
+
+* `registered_date_time` - The registration time of the worker machine.
```
Comment (katbyte) [inline website/docs/r/automation_hybrid_runbook_worker.html.markdown:49]: wrong tense?
```suggestion
* `registration_date_time` - The registration time of the worker machine.
```

**Review by katbyte — CHANGES_REQUESTED** [review]
Just a couple comments

File: website/docs/r/automation_hybrid_runbook_worker.html.markdown (hunk 4 above)
Comment (wuxu92) [reply website/docs/r/automation_hybrid_runbook_worker.html.markdown:49]: en...the property name from the swagger definition is `registeredDateTime`

[hybridRunbookWorker.json#L377](https://github.com/Azure/azure-rest-api-specs/blob/56eb34160d11aeff73c20a81d2a485bc0966a26e/specification/automation/resource-manager/Microsoft.Automation/stable/2021-06-22/hybridRunbookWorker.json#L377)

**Review by katbyte — APPROVED** [review]
Thanks @wuxu92 - LGTM 🌻

**Pull request merged**